package.json

```diff
@@ -11,7 +11,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "run-if-diff --file src/**/*.scss -- npm run build && git add dist/"
+      "pre-commit": "run-if-diff --file-path 'src/*.scss' -- npm run build && git add dist/"
```

Comment:
- updated option name from `--file` to `--file-path` due to addition of additional file filters (i.e. `--file-status`)
- quoted `*` to avoid interpretation by shell before execution of `run-if-diff`
- removed `**` since the glob is now interpreted by the executed `git` cmd and not `micromatch` (and as a consequence globbing syntax has slightly changed)